src/api/api.types.tsx

```diff
@@ -217,3 +217,23 @@ export interface Item
   usage_status: string;
   properties: Property[];
 }
+
+// ------------------------------------ ATTACHMENTS ------------------------------------------------
+
+export interface AttachmentsPostMetadata {
+  entity_id: string;
+  file_name: string;
+  title?: string | null;
+  description?: string | null;
+}
+
+export interface UploadInfo {
```

Comment:
```suggestion
export interface AttachmentPostUploadInfo {
```
Just aligns it with the backend like `CatalogueCategoryPostProperty` does above.